docs/changehistory/NextVersion.md

```diff
@@ -35,10 +36,24 @@ Table of contents:
 - `PropertyFilterBuilderToolbar` component to display the action buttons in the filter builder. #686
 - `isGroupOperatorDisabled` property of `PropertyFilterBuilderRendererProps` to control whether the group operator is toggle-able. #686
 
+### FilterBuilder changes
+
+Added support for `Between` and `Not between` operators in `PropertyFilterBuilder`. This required to make some breaking changes to `@beta` APIs of `PropertyFilterBuilder`. These changes include internal components of `PropertyFilterBuilder` (`PropertyFilterBuilderRuleValue` and `PropertyFilterBuilderActions`) that are exposed to allow customizing `PropertyFilterBuilder` component. If `PropertyFilterBuilder` is used without any customizations these changes should not break anything.
+
+- Updated UI of the `FilterBuilder` component. #686
+- Updated `PropertyFilterRule.operator` property type to support setting it using `PropertyFilterRuleOperator` enum or string value.
+- Changed `PropertyFilterBuilderRuleValue` component props to `PropertyFilterBuilderRuleValueRendererProps` in order to have access to rule operator when rendering value input.
+- Changed `PropertyFilterBuilderRule.operator` type from `PropertyFilterRuleOperator` to `PropertyFilterBuilderRuleOperator`.
+- Renamed `getPropertyFilterOperators` to `getPropertyFilterBuilderOperators`.
+- Renamed `getPropertyOperatorLabel` to `getPropertyFilterBuilderOperatorLabel`.
+- Renamed `PropertyFilterBuilderRuleOperatorProps` to `PropertyFilterBuilderRuleOperatorRendererProps`.
+- Added `PropertyFilterBuilderRuleOperator` type that has `between` and `not-between` value in addition to `PropertyFilterRuleOperator` values.
```

Comment:
Seems like these should go under the `Additions` section.

Reply:
I wanted to keep all changes related to addition of `Between` operator in one group. But I can move them under different sections if you want.

Reply:
Please do, otherwise, we'll end up with all the changes under the `Changes` section.

Reply:
Moved in https://github.com/iTwin/appui/pull/702/commits/f56b07d9bf70e668418ffe577c9bda2524bfd61d